src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/PolicyHelpStrings.cs

```diff
@@ -0,0 +1,99 @@
+        public const string NewPolicySetDefinitionNameHelp = "The name of the new policy set definition.";
+        public const string NewPolicySetDefinitionDisplayNameHelp = "The display name for the new policy set definition.";
+        public const string NewPolicySetDefinitionDescriptionHelp = "The description for the new policy set definition.";
+        public const string NewPolicySetDefinitionMetadataHelp = "The metadata for the new policy set definition. This can either be a path to a file name containing the metadata, or the metadata as a string.";
+        public const string NewPolicySetDefinitionPolicyDefinitionHelp = "The policy definition for the new policy set definition. This can either be a path to a file name containing the policy definitions, or the policy set definition as a string.";
+        public const string NewPolicySetDefinitionParametersHelp = "The parameters declaration for the new policy set definition. This can either be a path to a file name or uri containing the parameters declaration, or the parameters declaration as a string.";
+        public const string NewPolicySetDefinitionManagementGroupHelp = "The name of the management group of the new policy set definition.";
+        public const string NewPolicySetDefinitionSubscriptionIdHelp = "The subscription ID of the new policy set definition.";
+        public const string RemovePolicySetDefinitionNameHelp = "The policy set definition name to delete.";
+        public const string RemovePolicySetDefinitionIdHelp = "The fully qualified policy set definition ID to delete, including the subscription or management group. e.g. /subscriptions/{subscriptionId}/providers/Microsoft.Authorization/policySetDefinitions/{policySetDefinitionName}.";
+        public const string RemovePolicySetDefinitionManagementGroupHelp = "The name of the management group of the policy set definition to delete.";
+        public const string RemovePolicySetDefinitionSubscriptionIdHelp = "The subscription ID of the policy set definition to delete.";
+        public const string SetPolicySetDefinitionNameHelp = "The name of the policy set definition to update.";
+        public const string SetPolicySetDefinitionIdHelp = "The fully qualified policy set definition ID to get, including the subscription or management group. e.g. /providers/Microsoft.Management/managementGroups/{managementGroup}/providers/Microsoft.Authorization/policySetDefinitions/{policySetDefinitionName}.";
+        public const string SetPolicySetDefinitionDisplayNameHelp = "The display name of the updated policy set definition.";
+        public const string SetPolicySetDefinitionDescriptionHelp = "The description of the updated policy set definition.";
+        public const string SetPolicySetDefinitionPolicyDefinitionHelp = "The policy definition of the updated policy set definition. This can either be a path to a file name containing the policy definitions, or the policy definition as a string.";
```

Comment:
use "definitions" (plural) in the help string

Reply:
Done. Similarly changed parameter => parameters